src/main/charts/bamboo-agent/Changelog.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.7.0
+
+**Release date:** 2022-10-24
+
+![AppVersion: 9.0.0](https://img.shields.io/static/v1?label=AppVersion&message=9.0.0&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* Pin Python version to 3.9.14 (#468)
+* Update Bitbucket ingress testing values (#466)
+* Ingress Class Name is moved under spec field. (#451)
+* CLIP-1520: Make ATL_FORCE_CFG_UPDATE configurable in values.yaml (#454)
```

Comment:
Please remove ticket number from description . As far as we have the PR# then we can trace it and no need to expose much info here.